dist/image-sequencer-ui.js

```diff
@@ -385,36 +385,41 @@ function DefaultHtmlStepUi(_sequencer, options) {
     step.ui =
       '\
       <div class="container">\
-    <div class="row step">\
-    <form class="input-form">\
-    <div class="col-md-4 details">\
-    <h3>\
-    <span class = "toggle">' +step.name + ' <i class="fa fa-caret-up toggleIcon" aria-hidden="true"></i></span>' +
-    '<span class="load-spin" style="display:none;"><i class="fa fa-circle-o-notch fa-spin"></i></span>' +
-    '</h3><div class="cal collapse in"><p><i>"'+
-      (step.description || "") +
-      '</i></p></div>\
-    </div>\
-    </form>\
-    <div class="col-md-8 cal collapse in step-column">\
-      <div class="load" style="display:none;"><i class="fa fa-circle-o-notch fa-spin"></i></div>\
-      <div class="step-image">\
-        <a class="cal collapse in"><img alt="" class="img-thumbnail step-thumbnail"/></a>\
-      </div>\
-    </div>\
-    </div>\
-    </div>\
-    </div>';
+        <form class="input-form">\
+          <div class="panel panel-default">\
+            <div class="panel-heading">\
+              <div class="trash"></div>\
+                <h3 class="panel-title">' +  
+                  '<span class="toggle">' +step.name + '<span class="caret"></span>\
+                </h3>\
+              </div>\
+            <div class="panel-body cal collapse in">\
+              <div class="row step">\
+                <div class="col-md-4 details">\
+                  <h3>\
+                    <span class="load-spin" style="display:none;"><i class="fa fa-circle-o-notch fa-spin"></i></span>' +
+                      '</h3><div class="cal collapse in"><p><i>"'+
+                        (step.description || "") +
+                      '</i></p></div>\
+                </div>\
+              <div class="col-md-8 cal collapse in step-column">\
+                <div class="load" style="display:none;"><i class="fa fa-circle-o-notch fa-spin"></i></div>\
+                  <div class="step-image">\
+                    <a class="cal collapse in"><img alt="" class="img-thumbnail step-thumbnail"/></a>\
```

Comment:
Do you change the alt text in the js code ?

Reply:
Mean???

Reply:
The alt attribute `alt=""` is not given any value. Is it given a value later in the js code? Btw, please rebase.

Reply:
No I didn't made any change!!!

Reply:
So I think that part can be removed. This one `alt=""`